tests/scripts/runtime_unit_test_case_stat.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# List of gtest binaries
+GTEST_BINS=(
+    Product/out/unittest_standalone/nnfw_api_gtest
+    Product/out/unittest_standalone/test_compute
+    Product/out/unittest_standalone/test_onert
+    Product/out/unittest_standalone/test_onert_backend_cpu_common
+    Product/out/unittest_standalone/test_onert_frontend_nnapi
+    Product/out/unittest_standalone/tflite_test
+)
```

Comment:
IMO

- Place this script in `infra/nnfw/command/` and rename `count-unittest`
- Don't use hard-coded `Product/out/` and pass by argument such as `--install-path Product/out`

Reply:
Updated as your comments.